### ISSUE TITLE
[EN][Quest][NorthRend][IceCrown]At The Enemy's Gates and A Valiant's Field Training

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4138]
#**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
**Couple issues to list**
1. There was a open report for not being able to trample the. Currently as of Jan 5 this is still bugged.
**Phasing issues**

1. If you go to far to the citadel you phase back to the old zone. Or completing the quest you phase back to the old zone.
2. This also effect A Valiant's Field Training due to the phasing which should apply here as well.

*Expected behaviour**: Tell us what should happen instead
You should be able to trample the  Boneguard Footman
Phasing:
The phasing area need to be expanded to the proper area.
As long as a player has this quest active he should stay phased to see the npc required for quest.

**Steps to reproduce the problem**:

1. Accept quest.
2. Complete quest area phases before handing in.
3. Run out of zone and back in to rephase

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
**At The Enemy's Gate**
[Video of Quest](https://www.youtube.com/watch?v=nP07iFgk-Aw)
[WoWHead Quest Link](http://www.wowhead.com/quest=13847/at-the-enemys-gates)
[WoWWiki Link](http://wowwiki.wikia.com/wiki/Quest:At_The_Enemy's_Gates_(Alliance))

**A Valiant's Field Training**
[WoWhead Quest Link](http://www.wowhead.com/quest=13760/a-valiants-field-training)
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**
![wowscrnshot_010517_204854](https://cloud.githubusercontent.com/assets/12252441/21704706/08bc9006-d389-11e6-9de9-79de1125251b.jpg)

![wowscrnshot_010517_204856](https://cloud.githubusercontent.com/assets/12252441/21704710/0e68beee-d389-11e6-8f49-dfe23fc49064.jpg)

![wowscrnshot_010517_210435](https://cloud.githubusercontent.com/assets/12252441/21704949/99572350-d38a-11e6-957f-8867f0654ba8.jpg)

**Comment by Rushor** [comment]
areas, that need the phase 4519 and 4522

currently it's only available for area 4519

**Comment by Rushor** [comment]
for A Valiant's Field Training everything works fine for me on ptr. fix for At The Enemy's Gates will be pushed now.

**Comment by Rushor** [comment]
DB/Quest: At The Enemy's Gates - Add phasing for Icecrown Area

* closes https://github.com/dalaranwow/dalaran-wow/issues/4138
* commit: 57dded87b359ce02adb71dab124bb127a0a93553

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TheRealestMcCoy** [comment]
So will this fix the part once you kill the last npc to complete the quest the area wont phase out?

**Comment by Rushor** [comment]
the fix for this creates phasing issue once icc is released, i will work on a solution

**Comment by Rushor** [comment]
@TheRealestMcCoy done.

DB/Quest: At The Enemy's Gates - Don't remove phasing auras on quest completion + Spawn ICC Phaseshifter that removes all Phasingauras in range of the Icecrown Citadel Entrance

* updates https://github.com/dalaranwow/dalaran-wow/issues/4138
* commit: 7982a0a59b9dadd312a79cbef3c065252a0c374f

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
DB/Quest: Battle Before The Citadel - Add Phasing for Icecrown Area

* updates https://github.com/dalaranwow/dalaran-wow/issues/4138
* commit: e74a5fc00c46b60f34437651f367ea846043badf

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
